src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py

```diff
@@ -770,11 +770,11 @@ def create_data_collection_endpoint(cmd, subscription, resource_group, region, e
 def validate_data_collection_settings(dataCollectionSettings):
     if 'interval' in dataCollectionSettings.keys():
         intervalValue = dataCollectionSettings["interval"]
-    if (bool(re.match(r'^[0-9]+[m]$', intervalValue))) is False:  # pylint: disable=used-before-assignment
-        raise InvalidArgumentValueError('interval format must be in <number>m')
-    intervalValue = int(intervalValue.rstrip("m"))
-    if intervalValue <= 0 or intervalValue > 30:
-        raise InvalidArgumentValueError('interval value MUST be in the range from 1m to 30m')
+        if (bool(re.match(r'^[0-9]+[m]$', intervalValue))) is False:  # pylint: disable=used-before-assignment
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=122958543&view=results) to validate the change.

- test_aks_create_with_private_cluster_with_monitoring_aad_auth_msi_with_ampls_with_highlogscale

Reply:
> Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=122958543&view=results) to validate the change.
> 
> * test_aks_create_with_private_cluster_with_monitoring_aad_auth_msi_with_ampls_with_highlogscale

> @FumingZhang , thanks for triggering the tests. looks like failing checks not related to this change, can you please take a look at it.

Reply:
requeued CI